Fixed #34983 -- Deprecated django.utils.itercompat.is_iterable(). (#17498)
django/django: docs/releases/5.1.txt

  ``ModelAdmin.log_deletions()`` and  ``LogEntryManager.log_actions()``
  instead.

> * The ``django.utils.itercompat.is_iterable()`` function is deprecated. Use
>   ``isinstance(..., collections.abc.Iterable)`` instead. The module
>   ``django.utils.itercompat`` module will also be removed as this is the last
>   function remaining.

Reviewer: ```suggestion
* The undocumented ``django.utils.itercompat.is_iterable()`` function and the
  ``django.utils.itercompat`` module are deprecated. Use
  ``isinstance(..., collections.abc.Iterable)`` instead.
```
